f8a_report/monthly_stack_report_main.py

```diff
@@ -0,0 +1,29 @@
+"""Monthly stacks report."""
+
+import logging
+from datetime import datetime as dt, timedelta, date
+from report_helper import ReportHelper
+from v2.report_generator import StackReportBuilder
+
+logger = logging.getLogger(__file__)
+
+
+def main():
+    """Generate the monthly stacks report."""
+    report_builder_v2 = StackReportBuilder(ReportHelper)
+    today = dt.today()
+
+    # Generate a monthly venus report
+    logger.info('Monthly Job Triggered')
+    last_day_of_prev_month = date(today.year, today.month, 1) - timedelta(days=1)
+    last_month_first_date = last_day_of_prev_month.strftime('%Y-%m-01')
+    last_month_end_date = last_day_of_prev_month.strftime('%Y-%m-%d')
+
+    # Monthly Report for v2
+    logger.info(f'Generating Monthly report v2 from '
+                f'{last_month_first_date} to {last_month_end_date}')
```

Comment:
Usually preformatted strings should avoided for logging. Because if a particular log level is disabled then the preformatted string becomes useless. 
Following is the ideal logging pattern
```
loggger.info('msg %s %s', val1, val2)
```

Reply:
Ok